app/src/main/java/com/teamagam/gimelgimel/app/common/logging/LoggerFactory.java

```diff
@@ -17,10 +20,12 @@
             new NativeLogger(LoggerFactory.class.getSimpleName());
 
     private static FileWriter sLogWriter;
+    private static String sExternalStorageDirectoryPath;
 
     public static void init(Context context) {
         try {
             sLogWriter = DiskLogger.createLogfileWriter(context);
+            sExternalStorageDirectoryPath = getExternalStorageDirectoryPath(context);
         } catch (DiskLogger.LogfileCreationException ex) {
             sInnerLogger.e("Initializing disk-logger failed", ex);
```

Comment:
This try-catch will make both disk-logger fail in case just one of them fails.
This doesn't need a fix because the old disk logger shouldn't be used anymore.
It just needs to be adjusted to the new log4j disk logger.